src/Pim/Bundle/CatalogBundle/Resources/config/cursors.yml

```diff
@@ -11,3 +11,13 @@ services:
             - '%akeneo_elasticsearch.cursor.cursor.class%'
             - '%pim_catalog.factory.product_cursor.page_size%'
             - 'pim_catalog_product'
+
+    pim_catalog.factory.product_bounded_cursor:
+        class: '%akeneo_elasticsearch.cursor.bounded_cursor_factory.class%'
+        arguments:
+            - '@akeneo_elasticsearch.client'
+            - '@pim_catalog.object_manager.product'
```

Comment:
Not sure we still need this manager and not just `doctrine.entity_manager`.
Maybe not to do here but in the "Drop MongoDB" PR of JJ.